sdk/modules/ui/cpp/include/metavision/sdk/ui/utils/opengl_api.h

```diff
@@ -21,9 +21,15 @@ inline int glewInit(void) {
     return GLEW_OK;
 }
 #else // OpenGL
+#if defined(__APPLE__) && !defined(__linux__)
+#include <OpenGL/gl.h>
+#include <OpenGL/glu.h>
+#include <OpenGL/gl3.h>
+#elif defined(__linux__)
```

Comment:
We still want to include the below files for windows.

Reply:
I believe my most recent commit addresses most of your OpenGL related concerns.
I was also able to validate a working build on Windows. Thanks for your inputs. Hopefully it's improved and headed in the right direction.